loc/models.py

```diff
@@ -238,21 +203,13 @@ class Meta:
     # Attributes that map to keys used by Lob's verifications API:
     LOB_ATTRS = ['primary_line', 'secondary_line', 'urbanization', 'city', 'state', 'zip_code']
 
-    def is_populated(self) -> bool:
-        '''
-        Return whether the model contains enough filled-out fields to be a useful
-        substitute to the address string.
-        '''
-
-        return bool(self.primary_line and self.city and self.state and self.zip_code)
-
     def as_lob_params(self) -> Dict[str, str]:
         '''
         Returns a dictionary representing the address that can be passed directly
         to Lob's verifications API: https://lob.com/docs#us_verifications_create
         '''
 
-        if not self.is_populated():
+        if not self.is_address_populated():
```

Comment:
I've renamed this method to make it less potentially ambiguous in subclasses.  It actually _is_ a bit confusing in `AddressDetails` since this model actually has a separate field called `address` but this shouldn't be the case for most subclasses.